### PR TITLE
Work-function-QE support

### PR DESCRIPTION
**Pull request by nfaguirrec**
Added functions to extract and plot results from work-function calculations made with QE-AMS -- SO103, SCMSUITE-9391

### CODE REVIEW

File: interfaces/adfsuite/ams.py [hunk 1]
```diff
@@ -1849,6 +1849,78 @@ def get_density_along_axis(
         z = (bin_edges[:-1] + bin_edges[1:]) / 2.0
         return z, density
 
+    def get_work_function_results(
```
Comment (dormrod) [inline interfaces/adfsuite/ams.py:1852]: Is it worth it to add a small rkf file and unit test this method?

E.g. https://github.com/SCM-NV/PLAMS/blob/trunk/unit_tests/test_amsresults.py#L21

File: tools/plot.py [hunk 2]
```diff
@@ -353,3 +351,132 @@ def plot_msd(job, start_time_fit_fs=None, ax=None):
     ax.set_title("MSD: Diffusion coefficient = {:.2e} m^2/s".format(diffusion_coefficient))
 
     return ax
+
+
+def plot_work_function(
```
Comment (dormrod) [inline tools/plot.py:356]: Does this need an example update to include this?

**Review by dormrod — COMMENTED** [review]
Does this need an example update

File: interfaces/adfsuite/ams.py (hunk 1 above)
Comment (nfaguirrec) [reply interfaces/adfsuite/ams.py:1852]: There are two tests, but in AMS: $AMSHOME/examples/QE/WorkFunction/, $AMSHOME/examples/QE/WorkFunction/DipoleCorrection_WorkFunction

File: tools/plot.py (hunk 2 above)
Comment (nfaguirrec) [reply tools/plot.py:356]: There are two tests, but in AMS: $AMSHOME/examples/QE/WorkFunction/, $AMSHOME/examples/QE/WorkFunction/DipoleCorrection_WorkFunction

**Review by dormrod — APPROVED** [review]

**Pull request merged**